packages/runtime/datastore/src/remoteChannelContext.ts

```diff
@@ -214,11 +236,21 @@ export class RemoteChannelContext implements IChannelContext {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         for (const message of this.pending!) {
             try {
-                this.services.deltaConnection.process(message, false, undefined /* localOpMetadata */);
+                switch (message.type) {
```

Comment:
I think if we force load channel on any rebase op, then we do not need these changes - keeps code simpler